sales_billing/constants.py

```diff
@@ -0,0 +1,38 @@
+CUSTOM_FIELDS = {
+    "Payment Entry": [
+        {
+            "fieldname": "column_break_42",
+            "fieldtype": "Column Break",
+            "insert_after": "get_outstanding_orders",
+        },
+        {
+            "depends_on": 'eval:doc.docstatus == 0 && doc.payment_type == "Receive" && doc.party_type == "Customer" && doc.party',
+            "fieldname": "get_invoices_from_sales_billing",
+            "fieldtype": "Button",
+            "insert_after": "column_break_42",
+            "label": "Get Invoices from Sales Billing",
+        },
+        {
+            "fieldname": "sales_billing",
+            "fieldtype": "Link",
+            "insert_after": "get_invoices_from_sales_billing",
+            "label": "Sales Billing",
+            "options": "Sales Billing",
+        },
+        {
+            "fieldname": "section_break_44",
+            "fieldtype": "Section Break",
+            "insert_after": "sales_billing",
+        },
+    ],
+}
+
+
+PROPERTY_SETTERS = {
+    "Withholding Tax Cert": [
+        (None, "default_print_format", "Withholding Tax Cert", "Data"),
+    ],
+    "Account": [
+        ("account_type", "options", "\nAccumulated Depreciation\nAsset Received But Not Billed\nBank\nCash\nChargeable\nCapital Work in Progress\nCost of Goods Sold\nDepreciation\nEquity\nExpense Account\nExpenses Included In Asset Valuation\nExpenses Included In Valuation\nFixed Asset\nIncome Account\nPayable\nReceivable\nRound Off\nStock\nStock Adjustment\nStock Received But Not Billed\nService Received But Not Billed\nTax\nTemporary\nCurrent Assets\nNon-current Assets\nCurrent Liabilities\nNon-current Liabilities", "Text"),
+    ],
```

Comment:
Sorry, I think we don't need these lines.
Note: จริงๆพี่ก็งงอยู่ว่ามันติดมาได้ไง เพราะ Withholding Taxt Cert มันเป็นของ thai_tax สงสัยจะผิดมาตั้งแต่ต้น